tutorials/05_embedding_net.ipynb

```diff
@@ -339,7 +339,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      " Neural network successfully converged after 96 epochs."
+      " Neural network successfully converged after 220 epochs."
```

Comment:
Just for the future, at best, don't commit the notebooks if you run them. 

If you want you can revert this change (but here it doesnt really matter)